public_html/wp-content/mu-plugins/blocks/assets/src/speakers/block-content.js

```diff
@@ -118,18 +150,29 @@ class SpeakersBlockContent extends Component {
 												<a
 													className={ 'wordcamp-speaker-session-link' }
 													href={ session.link }
+													// is there an esc_url() equivalent in g? or do we not need it because we're returning an Element, which gets escaped automatically?
 												>
 													{ decodeEntities( session.title.rendered.trim() ) || __( '(Untitled)', 'wordcamporg' ) }
 												</a>
 												<br />
 												<span className={ 'wordcamp-speaker-session-info' }>
 													{ session.session_track.length &&
+														// will this fatal similar to https://meta.trac.wordpress.org/changeset/8025/ ?
+														// if so, it'd be good to check for other instances of that problem throughout the plugin
+
 														sprintf(
 															/* translators: 1: A date; 2: A time; 3: A location; */
 															__( '%1$s at %2$s in %3$s', 'wordcamporg' ),
 															session.session_date_time.date,
 															session.session_date_time.time,
 															get( find( tracks, ( value ) => { return parseInt( value.id ) === head( session.session_track ) } ), 'name' )
+															// this is pretty difficult for a human to parse when it's all written as a single line, can it be broken up into more digestable chunks?
+															// i don't think it needs to be a named function, but just spreading it across multiple lines would increase the readability a lot
+
+															// why use `head` instead of `session.session_track[0]` ?
+															// it's one extra thing to learn, but the benefit isn't obvious. is it useful because it does some extra error checking or something?
+															// if so, could that be avoided by an extra condition in the `{ session.session_track.length &&` bit above? maybe that's overkill and head() is easier, though.
+															// seems like it's unnecessary in es6 though? https://www.sitepoint.com/lodash-features-replace-es6/
```

Comment:
> why use `head` instead of `session.session_track[0]` ?

I think I did this because `session.session_track[0]` doesn't always exist. `head` always grabs the first item, regardless of the index.

Reply:
Ah, good point about `[0]` being empty but `[n]` not being. IIRC, the ES6 replacement for `head()` solves that, so it might be the best option regardless.